lib/base-container.js

```diff
@@ -39,6 +39,12 @@ export default class BaseContainer {
 		return driverHelper.waitTillPresentAndDisplayed( this.driver, this.expectedElementSelector, this.explicitWaitMS );
 	}
 
+	waitRequestsToComplete() {
```

Comment:
this will only work on the customizer itself which is the only place where jQuery will be available, so it would be best to move this into the `lib/pages/customizer-page.js` to make sure it's not called from other pages.